package.json

```diff
@@ -13,7 +13,7 @@
     "url": "git+https://github.com/antfu/pkg-exports.git"
   },
   "bugs": "https://github.com/antfu/pkg-exports/issues",
-  "keywords": [],
+  "keywords": ["package", "export", "pkg"],
```

Comment:
```suggestion
```

We could probably just remove it

Reply:
code revert: `keywords` have been deleted.